rename classes and create shared class for fast-forward merge action (#857)
VirtusLab/git-machete-intellij-plugin: CONTRIBUTING.md

## PRs & releases

The base branch of the repository is `master`, but each regular (non-hotfix, non-release, non-backport) PR must be merged to `develop`. <br/>

> Due to the fact that base branch is not `develop`, merged PRs do not close linked issues (you have to do it manually). <br/>

Reviewer: ```suggestion
Due to the fact that the default branch is not `develop`, merging of PRs does not close linked issues (you have to close the issues manually). <br/>
```
